examples/node-typescript-app/tsconfig.json

```diff
@@ -13,6 +13,10 @@
     "noImplicitThis": true,
     "alwaysStrict": true,
     "jsx": "react",
-    "allowSyntheticDefaultImports": false
+    "allowSyntheticDefaultImports": false,
+    "baseUrl": ".",
+    "paths": {
+      "*": ["*", "libs/*"]
```

Comment:
Why is this needed?

Reply:
This makes is so that editors can natively resolve the module.